caluma/caluma_workflow/models.py

```diff
@@ -236,3 +249,16 @@ class Meta:
             GinIndex(fields=["assigned_users"]),
             GinIndex(fields=["meta"]),
         ]
+
+
+@receiver(post_init, sender=WorkItem)
+def set_name_and_description(sender, instance, **kwargs):
+    """
+    Ensure WorkItem has a name and description set.
+
+    Default to values from Task.
+    """
+    if not any((value for _, value in instance.name.items())):
```

Comment:
If you're not interested in the key of a dict, you can use `.values()`, turning this line into `if not any(instance.name.values())`

Reply:
Also you coud *probably* just evaluate `instance.name` in boolean context, which will return `False` if the dict is empty. Not sure if LocalizedField populates the keys by default right now, so we'd have to test this idea

Reply:
Great catch. Now using `.values()`. `bool(instance.name)` doesn't work.